File: snippets/syntax-common.js
const flightPMs = [
  { name: 'Theon', team: 'Feature' },
  { name: 'Steven', team: 'Inventory' },
  { name: 'Aput', team: 'Hore' },
];

const result = flightPMs
  .map(p => p.name)
  .map(name => name.toUpperCase())
  .join(', ');

console.log(result);

File: snippets/syntax-common.java
String result = "";

for (int i = 0; i < FlightPMs.size(); i++) {
  ProductManager p = FlightPMs.get(i);

  result += p.getName();

  if (i != FlightPMs.size() - 1) {
    result += ", ";
  }
}

System.out.println(result);